### ISSUE TITLE
Failed to install org-ref: Package ‘emacs-26.1’ is unavailable

### ISSUE THREAD

**Issue by castle-bravo** [jkitchin/org-ref#722]
When attempting to installing org-ref on emacs 25, I get the following error message:

`Failed to install org-ref: Package ‘emacs-26.1’ is unavailable`

In the help page for org-ref, only emacs-25.1 is listed as a requirement. This could be an issue with org-ref or one of the other requirements. I appreciate any suggestions to help me unsnarl this. Thanks.

**Comment by jkitchin** [comment]
I am not sure what is causing that. Are there any additional hints in the messages? Is that what happens when you try manually installing it?

**Comment by castle-bravo** [comment]
Hey John, I was able to get org-ref working by upgrading to Emacs 26. I'm afraid that I can no longer reproduce this. Prior to seeing this error, Emacs complained that the specific version of biblio-core required by org-ref was not available, but I was able to move on by manually installing biblio core. I didn't save the error messages, so I can't show you now. I'll try spinning up a fresh Ubuntu 18.04 VM and seeing if this issue can be reproduced.
